samples/routing/fowarding-address/sample.md

```diff
@@ -62,17 +62,17 @@ snippet: new-handler
 
 ### NServiceBus.ForwardingAddress
 
-Contains the implementation logic for leaving a forwarding address. There are two behaviors in the pipeline to implement this behavior.
+Contains the implementation logic for specifying a forwarding address. This is done by specifying two behaviors in the pipeline.
```

Comment:
```suggestion
Contains the implementation logic for specifying a forwarding address. This is done by adding two behaviors in the pipeline.
```

The term specifying was repeated but in different contexts. I would use different terms here.